protected/humhub/modules/user/controllers/PasswordSecurityValidator.php

```diff
@@ -0,0 +1,125 @@
+<?php
+namespace humhub\modules\user\controllers;
+
+use yii\validators\Validator;
+

```

Comment:
Please add comments to the class and properties.

Is there any alternative class or composer package available for this validation?